packages/lab/src/skip-link/SkipLinks.tsx

```diff
@@ -0,0 +1,20 @@
+import { forwardRef, HTMLAttributes } from "react";
+import cx from "classnames";
+import { makePrefixer } from "@jpmorganchase/uitk-core";
+
+import "./SkipLinks.css";
+
+const withBaseName = makePrefixer("uitkSkipLinks");
+
+export const SkipLinks = forwardRef<
+  HTMLUListElement,
+  HTMLAttributes<HTMLUListElement>
+>((props, ref) => {
```

Comment:
This should be changed to a named function e.g. forwardRef(function SkipLinks(){})